docs/platform-api/instances/index.md

```diff
@@ -296,4 +282,32 @@ Example of Successful Response:
 }
 ```
 
+### PATCH /instances/{id}
+
+Update an already existing element instance.
+
+Use `PATCH /instances/{id}` to update the tags associated with an authenticated instance. Updating tags removes any existing tags, so if you want to keep existing tags and add to them, include all of your tags in the JSON body.
```

Comment:
Do you think it's worth adding something like example or use case before the section about updating tags? I wonder if this implies that path is only used for updating tags?